app/src/main/res/values/style.xml

```diff
@@ -19,4 +19,48 @@
         <item name="fontPath">fonts/Roboto-Thin.ttf</item>
     </style>
 
+
+    <style name="Kanshu.Light.DarkToolBar" parent="Theme.AppCompat.Light.NoActionBar">
```

Comment:
Is "AppTheme" still used at all? If not, then it should be removed, otherwise we have two base themes.

Is values-v21/style.xml still needed?